CONTRIBUTING.md

```diff
@@ -0,0 +1,130 @@
+
+# Contributing to TDP
+
+TDP is an open source project hosted on [GitHub](https://github.com/TOSIT-IO/TDP) administered by the [TOSIT association](https://tosit.fr/). It is release under the [Apache License 2.0](https://github.com/TOSIT-IO/TDP/blob/main/LICENSE).
+
+We appreciate your interest and efforts to contribute to TDP.
+
+## How to help
+
+Contributions go far beyond pull requests and commits. we are thrilled to receive a variety of other contributions including the following:
+
+- Write and publish your own actions
+- Write articles and blog posts, create a tutorial, and spread the words
+- Submit new ideas of features and documentation
+- Submitting documentation updates, enhancements, designs, or bugfixes
+- Submitting spelling or grammar fixes
+- Additional unit or functional tests
+- Help to answer questions and issues on GitHub
+
+## Getting help
+
+### Discussions
+
+There is currently no channel dedicated to discussing TDP. For now, you may simply open a new issue.
```

Comment:
We need to discuss about that. Issue is not the place to discuss, on github "discussions" exists for that. For me an issue is created for bug or for feature previously validated in discussions.

Reply:
I will do a proposal for the use of Github Discussions in another PR